dsc/tests/dsc_resource_list.tests.ps1

```diff
@@ -17,7 +17,7 @@ Describe 'Tests for listing resources' {
     }
 
     It 'dsc resource list --tags "<tags>" and --description "<description> work' -TestCases @(
-        @{ tags = 'linux'; description = $null; expectedCount = 1; expectedType = 'Microsoft/OSInfo' }
+        @{ tags = 'linux'; description = $null; expectedCount = 2; expectedType = @('DSC.PackageManagement/Apt', 'Microsoft/OSInfo') }
```

Comment:
Since this test now fails on Windows and macOS, you'll probably need to modify this test.  Simplest might be something like:

```powershell
expectedType = if ($IsLinux) { @(...) } else { @(...)}
```